examples/03-Running-on-multiple-GPUs-or-on-CPU.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>Can we move the description under <code>Running </code> on multiple GPUs?</p><p> </p><p>The headings are</p><p>- Downloading the dataset</p><p>-- MultiGPU and multi-node scaling</p><p>- Running in multiple-GPUs</p><p> </p><p>Why is multiGPU and multi-node scaling under the heading Downloading the dataset?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
done